src/api/app/controllers/webui/search_controller.rb

```diff
@@ -56,7 +56,7 @@ def search
       unless disturl_pkgrev.nil?
         disturl_rev, disturl_package = disturl_pkgrev.split('-', 2)
       end
-      if Package.exists_by_project_and_name(disturl_project, disturl_package, follow_multibuild: true)
+      if disturl_project.present? && disturl_package.present? && Package.exists_by_project_and_name(disturl_project, disturl_package, follow_multibuild: true)
```

Comment:
Looks like we need a disturl lib that parses such a dist url and returns a package object :thinking:

Reply:
better than moving more things into the package model :)

Reply:
That's why I said disturl lib :wink: